package.json

```diff
@@ -25,7 +25,6 @@
     "@rollup/plugin-typescript": "^11.0.0",
     "@types/jest": "^29.5.0",
     "@types/node": "^18.11.18",
-    "@types/react": "^18.2.20",
```

Comment:
These have moved to `packages/platforms/react-native` as they aren't needed by other packages